homepage/templates/homepage/add_review.html

```diff
@@ -0,0 +1,21 @@
+<html>
+<head>
+<title>Class Rater Add Review</title>
+<style type="text/css">
+    .author, .eom { color: grey }

```

Comment:
the style row (4-6) appear in every page - so maybe add them to the app_layout head area and remove from the rest of the files?

Reply:
Removed from the rest of the files